src/post/post.entity.ts

```diff
@@ -106,9 +107,29 @@ export class Post extends BaseEntity {
         if (!account.id) {
             throw new Error('Cannot add like for account with no id');
         }
+        this.likesToRemove.delete(account.id);
         this.potentiallyNewLikes.add(account.id);
     }
 
+    removeLike(account: Account) {
+        if (!account.id) {
+            throw new Error('Cannot add like for account with no id');
+        }
+        this.potentiallyNewLikes.delete(account.id);
+        this.likesToRemove.add(account.id);
+    }
+
+    getChangedLikes() {
+        const likesToRemove = [...this.likesToRemove.values()];
+        this.likesToRemove.clear();
+        const likesToAdd = [...this.potentiallyNewLikes.values()];
+        this.potentiallyNewLikes.clear();
+        return {
+            likesToRemove,
+            likesToAdd,
+        };
+    }
+
     getPotentiallyNewLikes() {
```

Comment:
Looks like this function is not used anymore. Let's remove this?

Reply:
Will do this with the cleanup commit!